python/private/BUILD.bazel

```diff
@@ -179,6 +179,11 @@ bzl_library(
     srcs = ["full_version.bzl"],
 )
 
+bzl_library(
+    name = "glob_excludes_bzl",
+    srcs = ["glob_excludes.bzl"],
+)
```

Comment:
@rickeylev does this need to express a dependency on `:util.bzl`?

Reply:
Yes, that would be most correct. Fixed.